src/server/game/Entities/Player/Player.cpp

```diff
@@ -26750,3 +26762,73 @@ void Player::SendSpellCategoryCooldowns()
 
     SendDirectMessage(cooldowns.Write());
 }
+
+void Player::_LoadAccountMounts(PreparedQueryResult result)
+{
+    if (result)
+    {
+        do
+        {
+            AddMount((*result)[0].GetUInt32(), (*result)[1].GetBool());
+        } while (result->NextRow());
+    }
+}
+
+bool Player::AddMount(uint32 spellId, bool isFavorite /*= false*/, bool isNew /*= false*/)
+{
+    SpellInfo const* spellInfo = sSpellMgr->GetSpellInfo(spellId);
```

Comment:
few unused variables here too. Is this spellid check needed or is the GetMount() check below enough ?